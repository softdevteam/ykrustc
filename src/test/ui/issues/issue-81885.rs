const TEST4: fn() -> _ = 42;
                  //~^ ERROR the type placeholder `_` is not allowed within types on item
                  //signatures

fn main() {
    const TEST5: fn() -> _ = 42;
                      //~^ ERROR the type placeholder `_` is not allowed within types on item
                      //signatures

}
