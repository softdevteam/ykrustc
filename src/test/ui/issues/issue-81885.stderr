error[E0121]: the type placeholder `_` is not allowed within types on item signatures
  --> $DIR/issue-81885.rs:1:22
   |
LL | const TEST4: fn() -> _ = 42;
   |                      ^ not allowed in type signatures

error[E0121]: the type placeholder `_` is not allowed within types on item signatures
  --> $DIR/issue-81885.rs:6:26
   |
LL |     const TEST5: fn() -> _ = 42;
   |                          ^ not allowed in type signatures

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0121`.
